distribution/lib/Standard/Base/0.0.0-dev/src/Network/HTTP/Request_Body.enso

```diff
@@ -40,6 +41,18 @@ type Request_Body
     ## Empty request body; used for GET
     Empty
 
+    ## PRIVATE
+       For `Request_Body.Form_Data url_encoded=False`, this returns `Nothing`,
+       because the content type must include a content boundary.
```

Comment:
I appreciate this comment, I was about to ask why it's Nothing in that case as that was unclear.